core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -229,7 +246,7 @@ public static void resizePool(final ThreadPoolExecutor pool, final IntSupplier m
    */
   public static void resizePool(final ThreadPoolExecutor pool, final AccumuloConfiguration conf,
       final Property p) {
-    resizePool(pool, () -> conf.getCount(p), p.getKey());
+    resizePool(pool, () -> conf.getCount(p), p.getKey() + ".pool");
```

Comment:
I'm not sure adding ".pool" is correct here. From what I can tell, this is used for a log message and it's using the configuration property for `p`, and I think the pool names are sufficiently divorced from the property names now.

Reply:
Removed in 3bcc3f8854